rio/frontend/json.ml

```diff
@@ -0,0 +1,15 @@
+(* open Yojson.Basic *)
+
+let rec policy_to_json (ast : Policy.t) =
+  match ast with
+  | Class c -> `Assoc [ ("class", `String c) ]
+  | Fifo sl -> `Assoc [ ("fifo", `List (List.map policy_to_json sl)) ]
+  | RoundRobin sl ->
+      let lst = List.map policy_to_json sl in
```

Comment:
In the other branches, you have inlined this `List.map ... sl` but here you have a separate variable name for it. Either is fine, but maybe make it consistent?